### PR TITLE
Adding managed disk conversion tests

### PR DESCRIPTION
**Pull request by suyogms**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@suyogms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: test/commands/arm/vm/arm.vm-convert-to-managed-disks.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        var cmd = util.format('vm convert-to-managed-disks %s %s --json', groupName, vmPrefix).split(' ');
+        testUtils.executeCommand(suite, retry, cmd, function(result) {
+          result.exitStatus.should.equal(0);
+          done();
+        });
+      });
+
+      it('Delete VM should work', function(done) {
+        this.timeout(vmTest.timeoutLarge);
+        var cmd = util.format('vm delete %s %s --json', groupName, vmPrefix).split(' ');
```
Comment (huangpf) [inline test/commands/arm/vm/arm.vm-convert-to-managed-disks.js]: The CI test stops at this part. I think you need to add a '-q' flag to force it.

Comment (huangpf) [inline test/commands/arm/vm/arm.vm-convert-to-managed-disks.js]: -q, --quiet                            quiet mode, do not ask for delete confirmation

Comment (suyogms) [inline test/commands/arm/vm/arm.vm-convert-to-managed-disks.js]: Yes, already working on it, I also need to include captured traces for delete. Will submit the iteration soon.

File: test/commands/arm/vm/arm.vm-convert-to-managed-disks.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    });
+    afterEach(function(done) {
+      suite.teardownTest(done);
+    });
+
+    describe('vm', function() {
+
+      it('create VM should work', function(done) {
+        this.timeout(vmTest.timeoutLarge);
+        vmTest.checkImagefile(function() {
```
Comment (huangpf) [inline test/commands/arm/vm/arm.vm-convert-to-managed-disks.js]: The integration test is complaining about vmSize. Please use the following function to get the vmSize
```
        vmTest.getVMSize(location, suite, function() {
          vmSize = VMTestUtil.vmSize;
```

File: test/recordings/arm-cli-vm-create-tests/arm_compute_vm_set_should_be_able_to_update_the_VM_size.nock.js [hunk 3]
```diff
@@ -6,8 +6,8 @@ exports.getMockedProfile = function () {
   var newProfile = new profile.Profile();
```
Comment (huangpf) [inline test/recordings/arm-cli-vm-create-tests/arm_compute_vm_set_should_be_able_to_update_the_VM_size.nock.js]: I think this test file shall be reverted.

**Comment by huangpf** [comment]
@amarzavery @balajikris Tests passed. Please help merge the PR. Thanks.

**Pull request merged**